appeals/web/src/server/lib/mappers/appeal.mapper.js

```diff
@@ -171,46 +136,59 @@ export async function initialiseAndMapAppealData(data, currentRoute, session) {
 					items: [
 						{
 							text: 'Change',
-							href: `${currentRoute}/change/case-procedure`
+							href: `${currentRoute}/change-appeal-details/case-procedure`
 						}
 					]
 				}
 			}
 		},
 		input: [
+			//TODO: Post MVP => Can only be Written for HAS - Only written rep is used for HAS, please add any new ones when we add more case types
```

Comment:
As above, this comment is unclear - I assume this is referring to the case procedure?

Reply:
I realise the object this is within is named `caseProcedure` but it's now unclear why this comment exists when there are procedure types defined below for written, hearing and inquiry